panther_analysis_tool/test_case.py

```diff
@@ -50,7 +50,7 @@ def __iter__(self) -> Iterator:
     def __len__(self) -> int:
         return len(self._data)
 
-    def get(self, arg: str, default: Any = None) -> Any:
+    def get(self, arg: Any, default: Any = None) -> Any:
```

Comment:
Linter is complaining about this:

```
panther_analysis_tool/test_case.py:53:4: W0221: Parameters differ from overridden 'get' method (arguments-differ)
```
Not 100% sure the fix, but this didn't help.

Reply:
pulling in the latest master will fix this 👍🏻

Reply:
Done!